### ISSUE TITLE
Bug about "aimbat-sac2pkl -p"

### ISSUE THREAD

**Issue by haibo-wang-seis** [pysmo/aimbat#91]
line 81 in sacpickle.py:
'        with open(zfile(picklefile+'.'+zipmode, 'rb')) as f:'
Since this variable 'picklefile' containing suffix about zipmode (bz2 or gz), so I think there should be:
'       with open(zfile(picklefile, 'rb')) as f:'

**Comment by smlloyd** [comment]
`.bz2` or `.gz` is appended to the picklefile variable both when reading and writing. So it's consistent, but perhaps a bit confusing or poorly documented - perhaps because the compression option is rarely used? Maybe @xlougeo can weigh in on this. Thanks for reporting this!

**Comment by xlougeo** [comment]
Yes the compression option is rarely used. It was initially there to save disk space but I think it's not a problem anymore. We could remove it in the future
